Assets/Scripts/Minigame/BrushTeeth.cs

```diff
@@ -80,14 +85,26 @@ private void OnTriggerEnter2D(Collider2D other)
         }
 
         // Plays animation after swiping an n(_repetitions)-amount of times to left and right
-        // After successfully starting the animation, the object will disappear
+        // After successfully starting the animation, the object and the helper sprites will disappear
         private void BrushTeethAnim()
         {
             if (_leftCheck >= _repetitions && _rightCheck >= _repetitions)
             {
+                // Activate animation...
                 _anim.SetBool(BrushTeethReqDone, true);
+                // Set the grabbed objects sprite to null...

```

Comment:
Yes, but what effect does this have? Don't describe the literal code, but what it **_does_** ;) 
E.g. "Hide the grabbed object", but that could be achieved by disabling the renderer aswell.